content/dictionary/database.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: database
-definition: ''
-perspectives: []
+definition: A database is an organized collection of structured information, or data, typically stored electronically in a computer system.
+sources: 
+- sourceurl: https://www.oracle.com/database/what-is-database.html
+perspectives: 
+- meaning: A database is different than a spreadsheet in that a database allows for a greater manipulation of data, allowing for more complex operations to be performed. The most popular model is the Relational Database Model.
```

Comment:
```suggestion
definition: A database is an organized collection of structured information, or data, typically stored electronically in a computer system. A database is different than a spreadsheet in that a database allows for a greater manipulation of data, allowing for more complex operations to be performed. The most popular model is the Relational Database Model.
sources: 
- sourceurl: https://www.oracle.com/database/what-is-database.html
perspectives: 
```
Since the added information is applicable to the entire definition and not just one role, I'd suggest adding it to the definition.